README.md

```
@@ -1 +1,73 @@
-# oci-backend
\ No newline at end of file
+<p align="center">
+  <a href="http://nestjs.com/" target="blank"><img src="https://nestjs.com/img/logo-small.svg" width="200" alt="Nest Logo" /></a>
+</p>
+
+[circleci-image]: https://img.shields.io/circleci/build/github/nestjs/nest/master?token=abc123def456
```

Markdownlint

Unused link or image reference definition: "circleci-image"
Link and image reference definitions should be needed

(MD053, link-image-reference-definitions)